### PR TITLE
DONT MERGE testing astyle

### PR DESCRIPTION
**Pull request by kain88-de**
This is the result of auto formatting with astyle

`astyle --style=google --indent-modifiers --align-pointers=type --align-reference=type --pad-header -s4 --lineend=linux --break-after-logical --convert-tabs`

It is not possible to use rules that are as fine grained as with clang-format but the code has less odd places using a variable line length.

### CODE REVIEW

**Comment by daschuer** [comment]
Did you reformat ALL files? 

clang-format:  +6,298 −5,237
astyle: +4,994 −4,812

A good sign!

File: src/analyserbeats.cpp [hunk 1]
```diff
@@ -63,22 +63,22 @@ bool AnalyserBeats::initialise(TrackPointer tio, int sampleRate, int totalSample
...
     m_bPreferencesOffsetCorrection = static_cast<bool>(
-        m_pConfig->getValueString(
-            ConfigKey(BPM_CONFIG_KEY, BPM_FIXED_TEMPO_OFFSET_CORRECTION)).toInt());
+                                         m_pConfig->getValueString(
+                                                 ConfigKey(BPM_CONFIG_KEY, BPM_FIXED_TEMPO_OFFSET_CORRECTION)).toInt());
     m_bPreferencesReanalyzeOldBpm = static_cast<bool>(
-        m_pConfig->getValueString(
-            ConfigKey(BPM_CONFIG_KEY, BPM_REANALYZE_WHEN_SETTINGS_CHANGE)).toInt());
+                                        m_pConfig->getValueString(
+                                            ConfigKey(BPM_CONFIG_KEY, BPM_REANALYZE_WHEN_SETTINGS_CHANGE)).toInt());
```
Comment (daschuer) [inline src/analyserbeats.cpp]: This is a "bit" long. What happen if you limit th line lengths?

Comment (kain88-de) [inline src/analyserbeats.cpp]: No change with a limit of 80 columns. `clang-format` is much better here.

**Comment by kain88-de** [comment]
> A good sign!

This is only because I didn't limit the line length. limiting the max number of columns in `astyle` gives similar stats. Though I personally prefer the the output of `clang-format`

File: src/analyserbeats.cpp (hunk 1 above)
Comment (daschuer) [inline src/analyserbeats.cpp]: What is the alignment pattern here? 
```
   m_bPreferencesReanalyzeOldBpm = static_cast<bool>(
                                        m_pConfig->getValueString(
                                            ConfigKey(BPM_CONFIG_KEY, BPM_REANALYZE_WHEN_SETTINGS_CHANGE)).toInt());
```

IMHO clang fails here as well: 
```
    m_bPreferencesReanalyzeOldBpm = static_cast<bool>(
            m_pConfig->getValueString(
                             ConfigKey(BPM_CONFIG_KEY,
                                       BPM_REANALYZE_WHEN_SETTINGS_CHANGE))
                    .toInt());
```

My eclipse: 
```
    m_bPreferencesReanalyzeOldBpm = static_cast<bool>(m_pConfig->getValueString(
            ConfigKey(BPM_CONFIG_KEY,
            BPM_REANALYZE_WHEN_SETTINGS_CHANGE)).toInt());
```

It looks like we reach the borders of auto formatter smartness here. 
I think developers help is required anyway: 
```
    ConfigKey reanalyzeKey(BPM_CONFIG_KEY, BPM_REANALYZE_WHEN_SETTINGS_CHANGE);
    m_bPreferencesReanalyzeOldBpm = 
            static_cast<bool>(m_pConfig->getValueString(reanalyzeKey).toInt());
```

Comment (daschuer) [inline src/analyserbeats.cpp]: we see here: --max-instatement-indent=40

**Pull request closed without merging**